spec/models/host_spec.rb

```diff
@@ -132,7 +133,7 @@
 
     context "#start" do
       before do
-        allow_any_instance_of(described_class).to receive_messages(:validate_ipmi    => nil)
+        stub_supports(described_class, :start)
```

Comment:
stubbing the method under test has me nervous

Introduced https://github.com/ManageIQ/manageiq/pull/21916 so we could stub ipmi and still test the functionality

Reply:
:+1: updated